### ISSUE TITLE
File uploads are broken when using `selenium 4.x`

### ISSUE THREAD

**Issue by symonk** [aerokube/selenoid#1150]
When performing file uploads via the `send_keys(...)` selenium 4.x is now dispatching requests to:

```
Command.UPLOAD_FILE: ('POST', "/session/$sessionId/file"), # old
Command.UPLOAD_FILE: ('POST', "/session/$sessionId/se/file"), # new
```

I suspect selenoid needs updated to reflect the `/se/`

attempting to do this with selenoid and `selenium 4.x` will result in:

```
E       selenium.common.exceptions.WebDriverException: Message: unknown command: unknown command: session/dad1aa5469e3357eebc5c0f52e3865bf/se/file
E       Stacktrace:
E       #0 0x564bc0056ac3 <unknown>
E       #1 0x564bbfb308f8 <unknown>
E       #2 0x564bbfb81294 <unknown>
E       #3 0x564bbfb80ff7 <unknown>
E       #4 0x564bbfb084c4 <unknown>
E       #5 0x564bbfb09370 <unknown>
E       #6 0x564bc008825e <unknown>
E       #7 0x564bc009dafa <unknown>
E       #8 0x564bc00891b5 <unknown>
E       #9 0x564bc009f4c8 <unknown>
E       #10 0x564bc007d95b <unknown>
E       #11 0x564bbfb08042 <unknown>
E       #12 0x7f3a5505a0b3 <unknown>
```

This doesn't account for selenium 4.0 being fully W3c spec compliant

**Comment by symonk** [comment]
duplicate of https://github.com/aerokube/selenoid/issues/1079

**Comment by vania-pooh** [comment]
@symonk we are going to fix this in the next version. But this exact API is not present in spec. Thus Selenium guys decided to change path from `/file` to `/se/file`.

**Comment by KevinLuc95** [comment]
@vania-pooh 
May I know when we will release the fix for it?

**Comment by vania-pooh** [comment]
@KevinLuc95 within the next weeks.

**Comment by aandryashin** [comment]
Fixed by d6fc603e4c50338d929f4a8bd285cee514d91fb7
